### PR TITLE
[HIRFUS-40] Querying the Transportation Database

### PR DESCRIPTION
**Pull request by VincentNjele**
- In this PR
- I have created an accident_detail.sql file 
- And I have added the related queries and results

### CODE REVIEW

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 1]
```diff
@@ -0,0 +1,146 @@
+
+
+
+/* 1. Number of people who survived   */
+SELECT  count(s.survival_status) FROM survival_table s WHERE s.survival_status = 1
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:5]: This query is giving a wrong answer. Hint:count should be COUNT (*) as accident_survivals where blah blah blah.

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+/*Number of survivors = 290   */
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:7]: This answer is incorrect

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+
+/* 2. Number of people who did not survived   */
+SELECT  COUNT(s.survival_status) FROM survival_table s WHERE s.survival_status = 0
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:11]: Same as question 1 above

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+/*Number of passengers that did not survive = 424   */
+
+
+/*3. Those who were <27 and embarked at Queenstown, I could not get those that embarked and Q and C in the same query because they are all in the same column  */
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:16]: Its in the same column. yes but it is possible to get those that embarked from Q and C

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+SELECT p.full_name, p.sex, t.embarkatiON_point FROM passengers_table p INNER JOIN trip_table t ON t.passenger_id = p.id WHERE p.age < 27
+AND t.embarkatiON_point = 'Q'
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:19]: hint for Q and C is embarkation_point = ('Q'|'S');

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+/*  Those that embarked at Southampton = 202  */
+
+
+
+/* 5. id and names of passsengers who paid > 100  */
+SELECT p.id, p.full_name  FROM passengers_table p
+INNER JOIN trip_table t ON t.passenger_id = p.id WHERE 
+t.trip_fare >100 AND p.age > 35 
+AND t.num_of_siblings_or_spouces > 0
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:128]: Good

File: submissions/vincentNjele/Database/accident_detail.sql [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+545,"Douglas, Mr. Walter Donald"
+582,"Thayer, Mrs. John Borland (Marian Longstreth Morris)"
+661,"Frauenthal, Dr. Henry William"
+699,"Thayer, Mr. John Borland"
+764,"Carter, Mrs. William Ernest (Lucile Polk)"
+857,"Wick, Mrs. George Dennick (Mary Hitchcock)"
+*/
+
+/* 5. id and names of passsengers who paid > 100: Yes they had siblings/spouses */
+
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Database/accident_detail.sql:145]: They are 9 in number.

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please effect requested changes.

**Review by ToluEnny24 — APPROVED** [review]

**Review by ToluEnny24 — APPROVED** [review]
Well done @VincentNjele

**Comment by VincentNjele** [comment]
Thank you

**Pull request merged**